ramls/titles.raml

```diff
@@ -5,176 +5,75 @@ version: v1
-            example: !include examples/titles/titles_get_200_response.json
+            description: |
+              An array of titles comprising the results of the query.
+              The array will be paged if its length exceeds the value set with the `count` query param, or the default `count` of 25 in its stead.
+              The included metadata gives us the total result count outside of the paged context.
+            type: titleCollection
+            example:
+              strict: false
+              value: !include examples/titles/titles_get_200_response.json
   post:
     description: Create a new Custom Title.
     headers:
       Content-Type:
         example: application/vnd.api+json
     body:
       application/vnd.api+json:
-        properties:
-          data:
-            description: Needed because of JSON API
-            type: object
-            required: true
-            properties:
-              type: string
-              attributes:
-                description: Needed because of JSON API
-                type: object
-                required: true
-                properties:
-                  name:
-                    description: |
-                      Name of the new title that is to be created.
-                    type: string
-                    example: "A New Title For You"
-                    required: true
-                  publicationType:
-                    description: Publication type for new title.
-                    type: string
-                    enum: ["All", "Audiobook", "Book", "Book Series", "Database", "Journal", "Newsletter", "Newspaper", "Proceedings", "Report", "Streaming Audio", "Streaming Video", "Thesis & Dissertation", "Website", "Unspecified"]
-                    example: "Book"
-                    required: true
-          included:
-            description: Resource used to create new title.
-            type: array
-            required: true
-            items: includedPackageId
-        example: !include examples/titles/titles_post_request.json
+        description: Create a new Custom Title.
+        type: titlePostRequest
+        example:
+          strict: false
+          value: !include examples/titles/titles_post_request.json
     responses:
       200:
         description: OK
         body:
           application/vnd.api+json:
-            example: !include examples/titles/titles_post_200_response.json
+            description: The server has successfully fulfilled the POST request.
+            type: title
+            example:
+              strict: false
+              value: !include examples/titles/titles_post_200_response.json
       400:
         description: Bad
```

Comment:
Is this left incomplete/still WIP?

Reply:
it was copied from original raml file 
![image](https://user-images.githubusercontent.com/37537790/47705408-26f58b80-dc2f-11e8-8948-19991eb711ae.png)
but you are right, it can be left like this. I have noticed that for invalid POST user receives 422 error code instead of 400 as it is in raml file. 
![image](https://user-images.githubusercontent.com/37537790/47705591-a08d7980-dc2f-11e8-92eb-ac1190a5ef07.png)
I will change the raml to return 422 code and add an example section. 
Thank you @sduvvuri-ebsco for the notice.

Reply:
👍